pkg/generators/openapi.go

```diff
@@ -676,9 +677,16 @@ func (g openAPITypeWriter) generateProperty(m *types.Member, parent *types.Type)
 		return nil
 	}
 	omitEmpty := strings.Contains(reflect.StructTag(m.Tags).Get("json"), "omitempty")
-	if err := g.generateDefault(m.CommentLines, m.Type, omitEmpty); err != nil {
+	def, err := g.generateDefault(m.CommentLines, m.Type, omitEmpty)
+	if err != nil {
 		return fmt.Errorf("failed to generate default in %v: %v: %v", parent, m.Name, err)
 	}
+	if def != nil {
+		g.Do("Default: $.$,\n", *def)
+		if enumType, isEnum := g.enumContext.EnumType(m.Type); isEnum && !enumType.IsValid(*def) {
```

Comment:
should this check be part of `generateDefault` implementation instead? Just wondering since it is called from multiple places and might be cleaner that way

Reply:
it would mean we would have to push even more parameters to `generateDefault`, I don't have a strong opinion.

Reply:
Also, I don't see `IsValid` being called in other places?